docs/pages/setup/reference/cli.mdx

```diff
@@ -1310,6 +1310,342 @@ $ tctl auth rotate --type=user --grace-period=200h
+| `--ignore-missing-roles`  |                  |                            | Ignore missing roles referenced in `--teams-to-roles`.  |
+
+#### Global flags
+
+These flags are available for all commands `--debug, --config`. Run
+`tctl help <subcommand>` or see the [Global Flags section](#tctl-global-flags).
+
+#### Examples
+
+Generate Github auth connector. Two role mappings are defined:
+  - members of `admin` team in `octocats` org will receive `access`, `editor` and `auditor` roles.
+  - members of `dev` team in `octocats` org will receive `access` role.
+
+The values for `--secret` and `--id` are provided by GitHub on [OAuth app](https://docs.github.com/en/developers/apps/building-oauth-apps/authorizing-oauth-apps) configuration page.
+
+```code
+$ tctl sso configure gh -r octocats,admin,access,editor,auditor -r octocats,dev,access --secret GH_SECRET --id CLIENT_ID
+INFO [CLIENT]    RedirectURL empty, resolving automatically.
+INFO [CLIENT]    RedirectURL set to "https://teleport.example.com:3080/v1/webapi/github/callback"
+kind: github
+metadata:
+  name: github
+spec:
+  client_id: CLIENT_ID
+  client_secret: GH_SECRET
+  display: ""
+  redirect_url: https://teleport.example.com:3080/v1/webapi/github/callback
+  teams_to_logins:
+  - logins:
+    - access
+    - editor
+    - auditor
+    organization: octocats
+    team: admin
+  - logins:
+    - access
+    organization: octocats
+    team: dev
+version: v3
+```
+
+Generate the configuration and immediately test it using `tctl sso test` command.
+
+```code
+$ tctl sso configure gh ... | tctl sso test
+```
+
+### tctl sso configure oidc
+
+Configure OIDC auth connector, optionally using a preset.
+
+The flag `--claims-to-roles` can be provided multiple times.
+
+```code
+$ tctl sso configure oidc --id=CLIENT_ID --secret=SECRET --claims-to-roles=... [<other-flags>]
+```
+
+#### Arguments
+
+This command accepts no arguments.
+
+#### Flags
+
+Mandatory flags: `--id`, `--secret`, `--claims-to-roles`. Other flags may be required depending on chosen preset.
+
+General flags:
+
+| Name                      | Description                                                                                                                               |
+|---------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| `-p`, `--preset`          | Preset. One of: `google`, `gitlab`, `okta`.                                                                                               |
+| `-n`, `--name`            | Connector name. Required, unless implied from preset.                                                                                     |
+| `-r`, `--claims-to-roles` | Sets claim-to-roles mapping using format `claim_name,claim_value,role1,role2,...`. Repeatable.                                            |
```

Comment:
Does "Repeatable" mean you specify a new `--claims-to-roles` flag, or add some sort of delimiter?

Reply:
It can be given multiple times: `tctl sso configure oidc -r groups,devs,access -r groups,admins,editor,auditor ...`.